.github/workflows/release.yaml

```diff
@@ -5,6 +5,18 @@ on:
     tags:
       - "release/*"
 
+  workflow_dispatch:
```

Comment:
This allows us to kick off the release workflow from the `make_release.sh` script.